app/forms/idv/api_image_upload_form.rb

```diff
@@ -34,39 +34,45 @@ def initialize(
     end
 
     def submit
-      form_response = validate_form
+      validate_form
 
-      client_response = nil
-      doc_pii_response = nil
+      self.document_response_validator = DocumentResponseValidator.new(
+        form_response:,
```

Comment:
the document_response_validator is only used if the form_response is successful ... do we need to pass in the form_response into the constructor? 🤔